### PR TITLE
Add Templates as Workspaces

### PR DESCRIPTION
**Pull request by samchungy**
Gives us a better experience while working with them

### CODE REVIEW

File: template/koa-rest-api/package.json [hunk 1]
```diff
@@ -21,11 +22,10 @@
     "@opentelemetry/propagator-b3": "^1.25.0",
     "@opentelemetry/sdk-node": "^0.52.0",
     "@seek/logger": "^6.0.0",
-    "aws-sdk": "^2.1039.0",
```
Comment (samchungy) [inline template/koa-rest-api/package.json:24]: Not used? otel seems to support v3.

File: template/koa-rest-api/package.json [hunk 2]
```diff
@@ -21,11 +22,10 @@
...
     "hot-shots": "^10.0.0",
     "koa": "^2.13.4",
     "koa-bodyparser": "^4.3.0",
-    "koa-compose": "^4.2.0",
```
Comment (samchungy) [inline template/koa-rest-api/package.json:28]: image
image

File: pnpm-lock.yaml
Comment (72636c) [inline pnpm-lock.yaml:1]: One thing with the old model is that our CI broke when a clean `init` without a lockfile broke.

Is it worth scripting up frequent, automerged lock file maintenance so we get a similar feedback loop?

File: template/express-rest-api/package.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
+  "name": "<%- serviceName %>",
```
Comment (72636c) [inline template/express-rest-api/package.json:2]: Should we put a `@seek/` scope out of paranoia?

I've seen us pay out bounties for "dependency confusion" even though I tend to disagree with the premise.

**Review by 72636c — COMMENTED** [review]
👏

**Review by 72636c — APPROVED** [review]

File: pnpm-lock.yaml
Comment (samchungy) [reply pnpm-lock.yaml:1]: image

It always installs the latest in the integration test for the templates so we're safe there! I tested a downgrade of zod

File: package.json [hunk 4]
```diff
@@ -34,6 +34,7 @@
     "format": "pnpm --silent skuba format",
     "lint": "pnpm --silent skuba lint && pnpm --silent lint:md",
     "lint:md": "remark --frail --quiet .",
+    "prepack": "pnpm --filter './template/**' exec rm -rf node_modules",
```
Comment (samchungy) [inline package.json:37]: https://github.com/pnpm/pnpm/pull/7301

Workaround hack for this

**Pull request merged**